src/http_util.cpp

```diff
@@ -95,5 +97,21 @@ worker_t::result_t disk_result(const http_request_t& request,
   return result;
 }
 
+
+uint8_t get_method_mask(const std::string& verb_list) {
+  uint8_t method_mask = 0;
+  std::istringstream iss(verb_list);
+  std::string verb;
+
+  while (std::getline(iss, verb, ',')) {
+    auto it = STRING_TO_METHOD.find(verb);
+    if (it != STRING_TO_METHOD.end()) {
+      method_mask |= it->second;
+    }
+  }
+
+  return method_mask;
```

Comment:
if the person configures with some stuff or nothing at all that makes sense, we shouldnt default to accepting no requests, there is no point in running the server with a mask of 0. so i would suggest:

```suggestion
  return method_mask ? method_mask : ALL_VERBS_MASK;
```